### DISCUSSION TITLE
ipxe documentation suggestion

### DISCUSSION THREAD

**Discussion by brcisna** [ipxe/ipxe#747] [Ideas]
Hello All,

Just thought be worth mentioning, with cellular connections for internet becoming very popular now,,and the onset of CGNAT mostly the standard for cellular connections  in 'the quick start guide' on ipxe.org. 
May be worth adding into this documentation the 'chain http://boot.ipxe.org/demo/boot.php'  command will fail even though the previous commands in the quick start looks correct.

This fails due to the fact with CGNAT no firewall ports will be accessible and hence the 'chain http://boot.ipxe.org/demo/boot.php' fails.
This may be mentioned in the ipxe documentation some place and I missed it. 
Of course the only real work around to this is setting up a vpn to a VPS that has a public ip address,,and set allow rules on firewall ,or pay for an static ip address through the cellular provider,,,which must be a 'business account' through the provider usually . 
Have recently switched to a cellular connection recently,,so this hit me. Had to set up a vpn to a public ip addressed VPS. remotely ( for an http ipxe boot to actually work) .

Thanks

**Comment by NiKiZe** [comment]
What exactly does fail and In what way?

CGNAT does not break connectivity.
The process only requires the client to be able to reach the web, same way as a browser.

Now if you want to host the server yourself and have it accessed from outside that is a different matter. But is out of scope for this project.

Also, do use IPv6.

**Comment by brcisna** [comment]
what am i missing here?

Edit: Both LAN and WAN interfaces on server does have ipv6 address also. 

Thanks


![ipxe-fail1](https://user-images.githubusercontent.com/47399290/190626054-917d749b-292e-454f-8768-0d96c0f14438.png)

**Reply by NiKiZe** [reply]
Did you read the shown URL?

**Reply by stappersg** [reply]
@brcisna is missing much. One thing is seeing that `http://ipxe.org/3e11623b` is an URL.

**Reply by NiKiZe** [reply]
So you have no working DNS?

**Comment by brcisna** [comment]
Ok I dont understand?
shown URL---- ,http://ipxe.org/3e11623b
Dont know how to decypher it,,I guess,..Im showing my stupidity,,,sorry 
that error says no dns servers,,, if i do the 'show dns' i do have a public dns number showing 

thanks

**Reply by stappersg** [reply]
URL is updated.

![afbeelding](https://user-images.githubusercontent.com/35434094/190631596-6fb8afb5-d2be-4b19-ab4f-6bdecd72d372.png)

**Comment by brcisna** [comment]
OK,,,

When i do 'show dns'   I do get a  return,,,as mentioned earlier,, 8.8.4 4   (google dns) 
So client is reaching a public dns
 
Ill just kick in the Wireguard / vpn to public VPS and this works OK. 

Thanks for helping.

**Reply by stappersg** [reply]
> Thanks for helping.

That is probably the longer version the _"Thanks"_ and the _"thanks"_ we can see above.
As I don't think that [previous reply](https://github.com/ipxe/ipxe/discussions/747#discussioncomment-3662060) is an _Yes, I have now a working setup. Thanks for helping._

@brcisna please state explicite that you do have a working setup, that iPXE works for you. And that the _"Thanks"_ is about work that has been done.

In case the _"Thanks for helping."_    was too early:
* Complete a complete different task.  ( Walking the dog,  washing dishes )
* Have a fresh look at the challenge you are facing.
* Try again.
* Report back.

**Comment by stappersg** [comment]
This comment as marker for _the original posted idea has been discussed and there is no need for further action_.

Yeah, it would be good thing to be able to flag a _"github.com discussion idea"_ as **processed**.
